packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -89,10 +94,6 @@ describe('ApmServer', function() {
 
   it('should report http errors', function(done) {
     var apmServer = new ApmServer(configService, loggingService)
-    configService.setConfig({
-      serverUrl: 'http://localhost:54321',
-      serviceName: 'test-service'
-    })
```

Comment:
Why is this removed? This test is for the case when the apm-server is unreachable!

Reply:
The same with two other instances below.

Reply:
Forgot to revert those, was testing IE bug. I will address those with other review comments.